core/src/main/scala/kafka/coordinator/transaction/TransactionCoordinator.scala

```diff
@@ -54,9 +54,11 @@ object TransactionCoordinator {
     // we do not need to turn on reaper thread since no tasks will be expired and there are no completed tasks to be purged
     val txnMarkerPurgatory = DelayedOperationPurgatory[DelayedTxnMarker]("txn-marker-purgatory", config.brokerId, reaperEnabled = false, timerEnabled = false)
     val txnStateManager = new TransactionStateManager(config.brokerId, zkUtils, scheduler, replicaManager, txnConfig, time)
-    val txnMarkerChannelManager = TransactionMarkerChannelManager(config, metrics, metadataCache, txnStateManager, txnMarkerPurgatory, time)
 
-    new TransactionCoordinator(config.brokerId, scheduler, producerIdManager, txnStateManager, txnMarkerChannelManager, time)
+    val logContext = new LogContext("[Transaction Coordinator " + config.brokerId + "] ")
```

Comment:
To maintain the idea of key/value pairs in a context, we can maybe change this to: `new LogContext(s"[Transaction Coordinator id=${config.brokerId}] ")